pkg/models/openpitrix/utils.go

```diff
@@ -673,6 +673,17 @@ func filterApps(apps []*v1alpha1.HelmApplication, conditions *params.Conditions)
 		return apps
 	}
 
+	// filter app by param app_id
```

Comment:
Should replace with `pkg/models/resources/v1alpha3/interface.go:DefaultList` in next version